shotover-proxy/src/transforms/cassandra/sink_cluster/mod.rs

```diff
@@ -171,6 +173,11 @@ impl CassandraSinkCluster {
             local_table: FQName::new("system", "local"),
             peers_table: FQName::new("system", "peers"),
             peers_v2_table: FQName::new("system", "peers_v2"),
+            system_keyspaces: [
+                Identifier::parse("system"),
+                Identifier::parse("system_schema"),
+                Identifier::parse("repair_history"),
```

Comment:
Did you mean to specify the `system_distributed` keyspace? `repair_history` is the table name.

Reply:
good catch!